src/components/Header/MarketplaceHeader.tsx

```diff
@@ -54,40 +43,40 @@ const MarketplaceHeader: React.FC = () => {
         history.push('/marketplace/wishList')
         break
       case 'Inventory':
-        if (!account) return popup({ success: false, message: t('connectWallet') })
-       history.push('/inventory/home/draft')
+        history.push('/inventory/home/draft')
```

Comment:
if not connect to wallet then it should to redirect to marketplace becauseof user not connected and enventory have no data